CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## NEXT VERSION
+
+  * **Bugfix: obfuscating URLs to external services no longer modifying original URI
```

Comment:
The markdown formatting needs to be fixed on this line.

Reply:
fixed!